<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageEditorContainer</class>
 <widget class="QWidget" name="PageEditorContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="addPageLabel">
        <property name="text">
         <string>Add page:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="addPageList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPageButton">
        <property name="toolTip">
         <string>Add new page</string>
        </property>
        <property name="icon">
         <iconset resource="page_editor.qrc">
          <normaloff>:/images/document-new.png</normaloff>:/images/document-new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePageButton">
        <property name="toolTip">
         <string>Remove current page</string>
        </property>
        <property name="icon">
         <iconset resource="page_editor.qrc">
          <normaloff>:/images/document-close.png</normaloff>:/images/document-close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clonePageButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Make a copy of current page</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="page_editor.qrc">
          <normaloff>:/images/document-multiple.png</normaloff>:/images/document-multiple.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ItemsToolWidget" name="tools" native="true"/>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBar" name="actions">
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="pageLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="FancyTabWidget" name="pageTabs" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="ObjectInspectorLayout"/>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="PropertyEditorLayout"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>frame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FancyTabWidget</class>
   <extends>QWidget</extends>
   <header location="global">fancytabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemsToolWidget</class>
   <extends>QWidget</extends>
   <header>itemstoolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="page_editor.qrc"/>
 </resources>
 <connections/>
</ui>
